addons/intercom_phone/CfgVehicles.hpp

```diff
@@ -0,0 +1,16 @@
+class CfgVehicles {
+    #include "ACEActions.hpp"
+
+    class LandVehicle;
+    ADD_ACTIONS(LandVehicle, Car)
+    ADD_ACTIONS(LandVehicle, Tank)
+    ADD_ACTIONS(LandVehicle, Motorcycle)
```

Comment:
Do Motorcycle's ever have intercoms? :D 

I'd say that can be skipped, more important here would be helicopters I'd say